### ISSUE TITLE
Give collective admins and fiscal host admins root admin 

### ISSUE THREAD

**Issue by shannondwray** [opencollective/opencollective#6220]
## Who is your user?
Collective admin or Fiscal Host admin 

## What are they trying to achieve?
Can we give them access to their own version of root admin so they can do things like move contributions etc

_When I want to perform any of the root admin actions, I want to not have to contact support, so I can have more autonomy over my account <outcome>_

## How are they currently doing this?
Currently have to contact support and a team member with admin access to Open Collective can action the moving of an expenses etc. 

## How well understood is this problem?

 - [ ] Surfacing: if you feel this is a seed of an idea that needs to be researched.
 - [ ] Understanding: if you've done some research and feel it is already well understood and ready for shaping.
 - [ ] Shaping:  if you feel a solution is clearly defined and can be made into a pitch ([read more about shaping](https://basecamp.com/shapeup/1.1-chapter-02))
 - [ ] Pitching: if you feel that it is ready to be presented for prioritisation ([read more about pitching](https://basecamp.com/shapeup/1.5-chapter-06))

**Comment by Betree** [comment]
The core principle of the root admin is that it includes tools that only root admins should have access to. Take the ban accounts for exemple, we don't want anyone else to start banning users on our platform.
It's a way for engineering to quickly implement actions that will help the support team without paying any attention to the design and with limited implications on security.

"giving host/collective admins" access to the root admin doesn't make much sense in this context: if there's a feature we want to expose for them, it just needs to be properly designed to include it in their existing admin panel (/collective/admin). That's what we've done so far.

**Comment by shannondwray** [comment]
Yeah I agree! I feel like moving contributions and expenses could be something that we give collective and fiscal host admins the ability to do :)
